test/functional/server.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test RPC calls related to the server.
+
+Tests correspond to code in rpc/server.cpp.
+"""
+
+import time
+
+from test_framework.test_framework import BitcoinTestFramework
+
+
+class ServerTest(BitcoinTestFramework):
+    def __init__(self):
+        super().__init__()
+
+        self.num_nodes = 1
+        self.setup_clean_chain = True
+
+    def run_test(self):
+        self._test_uptime()
+
+    def _test_uptime(self):
+        # The number of seconds to wait before checking the uptime value
+        wait_time = 6
+
+        # Depending on how fast the setup executes, the RPC server starts and this test
+        # executes the check for the uptime value might not deterministic. At the very
+        # least we will check if the reported uptime is the amount of seconds we waited.
+        time.sleep(wait_time)
```

Comment:
Arghh I didn't know about this `setmocktime`. Didn't think of searching either. Will get to it.